cx/spark/README.md

```diff
@@ -0,0 +1,50 @@
+# Spark Integration for Dialogflow CX
+
+## Setup
+
+### Prerequisites
+
+- Follow the instructions on the [main README file](https://github.com/GoogleCloudPlatform/dialogflow-integrations#readme) in the root directory of this repository.
+- Create a [Spark account](https://teams.webex.com/signin).
+- Uncomment and fill in the variables on the top of the [server.js file](https://github.com/rdouglas202/dialogflow-integrations/blob/b56f126598fb8128bd8f22d3b7342345e5ca821d/cx/spark/server.js#L16).
```

Comment:
It might be better to link the file in this repository instead of your personal repository.

E.g., https://github.com/GoogleCloudPlatform/dialogflow-integrations/blob/9df026cdb76aeb2d664767ca36f5f84ca4cb71b6/cx/spark/server.js